### PR TITLE
enable Status check assertions for sst_file_reader_test

### PR DESCRIPTION
**Pull request by ajkr**

### CODE REVIEW

File: table/sst_file_writer.cc [hunk 1]
```diff
@@ -135,25 +133,30 @@ struct SstFileWriter::Rep {
...
 
-    InvalidatePageCache(false /* closing */);
-
-    return Status::OK();
+    return InvalidatePageCache(false /* closing */);
   }
 
-  void InvalidatePageCache(bool closing) {
+  Status InvalidatePageCache(bool closing) {
+    Status s;
```
Comment (mrambacher) [inline table/sst_file_writer.cc:140]: Please initialize s = Status::OK(); for clarity

**Review by mrambacher — APPROVED** [review]
LGTM.

File: table/sst_file_writer.cc (hunk 1 above)
Comment (ajkr) [reply table/sst_file_writer.cc]: Done.

**Review by ajkr — COMMENTED** [review]
Thanks for the review!

**Pull request closed without merging**